src/styles/components/typography.less

```diff
@@ -196,6 +196,14 @@ components/typography.less
 
   }
 
+  .text-overflow-break-word {
+
+    overflow-wrap: break-word;
+    word-wrap: break-word;
+    word-break: break-word;
```

Comment:
This is my source of reference: https://css-tricks.com/snippets/css/prevent-long-urls-from-breaking-out-of-container/

From their explanation, I understand that all those are necessary.